test/ng-build/ng5/src/app/app.module.ts

```diff
@@ -13,12 +13,13 @@ import { AppComponent } from './app.component';
   ],
   imports: [
     BrowserModule,
-    AngularFireModule.initializeApp({}),
     AngularFireAuthModule,
     AngularFireDatabaseModule,
     AngularFirestoreModule
   ],
-  providers: [],
+  providers: [
```

Comment:
Seeing this in action, I feel like it increases the "getting started" complexity. We should continue to mirror the Firebase API rather.

Reply:
Yeah I was thinking this too. I'll bring `AngularFireModule` back in and allow users to use both whichever fits their use case best. Say in a specific module they need to call out to another app/database/whatnot.